api/v1alpha1/validation.go

```diff
@@ -855,6 +856,16 @@ func ValidateApplicationContent(content []byte, appType AppType) []error {
 			return []error{fmt.Errorf("parse compose spec: %w", err)}
 		}
 		allErrs = append(allErrs, validation.ValidateComposeSpec(composeSpec)...)
+	case AppTypeQuadlet:
+		// Quadlet apps can come with misc files, so only validate that the quadlet files are defined correctly
```

Comment:
question: what is the expectation for these misc files. can you give an example? lets make sure docs are eventually solid around this.